Ip.py

```diff
@@ -0,0 +1,58 @@
+"""The extension finds out your internal and external ip address"""
+
+from shutil import which
+from albertv0 import *
+import subprocess
+import socket
+import os
+
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "IP Addresses"
+__version__ = "1.0"
+__trigger__ = "ip "
+__author__ = "Benedict Dudel"
+__dependencies__ = ["curl"]
+
+if which("curl") is None:
+    raise Exception("'curl' is not in $PATH.")
+
+iconPath = iconLookup("preferences-system-network")
+
+
+def handleQuery(query):
+    if not query.isTriggered:
+        return None
+
+    proc = subprocess.run(["curl", "ipecho.net/plain"], stdout=subprocess.PIPE)
```

Comment:
Please use urllib for that to avoid the dependencies. Afaik curl is not always available.